src/System.Management.Automation/security/SecuritySupport.cs

```diff
@@ -417,7 +417,7 @@ public static bool IsProductBinary(string file)
             // On OneCore SKUs like NanoServer/IoT, the API has a bug that makes it not able to find the
             // corresponding catalog file for a given product file, so it doesn't work properly.
             // In these cases, we just trust the 'isUnderProductFolder' check.
-            if (Signature.CatalogApiAvailable.HasValue && !Signature.CatalogApiAvailable.Value)
+            if (!Signature.CatalogApiAvailable.GetValueOrDefault())
```

Comment:
We shouldn't change current logic:
```suggestion
            if (Signature.CatalogApiAvailable.HasValue && !Signature.CatalogApiAvailable.GetValueOrDefault())
```